.github/workflows/makefile.yml

```diff
@@ -0,0 +1,17 @@
+---
+name: Linting
```

Comment:
This is really only markdown linting. Projects will still need to define their own linters